web_plugin/src/main/java/com/futureworkshops/mobileworkflow/plugin/web/view/webview/LoggerWebChromeClient.kt

```diff
@@ -32,10 +35,31 @@ fun ConsoleMessage.logMessage(
 
 open class LoggerWebChromeClient(
     private val context: Context,
-    private val logger: Logger
+    private val logger: Logger,
+    private val activity: Activity?
 ): WebChromeClient() {
+    private var customView: View? = null
+
     override fun onConsoleMessage(consoleMessage: ConsoleMessage?): Boolean {
         consoleMessage?.logMessage(context, logger)
         return true
     }
+
+    override fun onHideCustomView() {
+        activity?.let {
+            (it.window.decorView as FrameLayout).removeView(customView)
+            customView = null
+        }
+    }
+
+    override fun onShowCustomView(paramView: View, paramCustomViewCallback: CustomViewCallback) {
+        if (customView != null) {
+            onHideCustomView()
+            return
```

Comment:
@tomermandelfw Here, if there is already a custom view, you are removing the old one and finishing. So, the new `paramView` is not added. Is this the desired behaviour?